Makefile

```diff
@@ -145,11 +145,11 @@ ubi-image-nap-plus: build ## Create Docker image for Ingress Controller (UBI wit
 
 .PHONY: ubi-image-dos-plus
 ubi-image-dos-plus: build ## Create Docker image for Ingress Controller (UBI with NGINX Plus and NGINX App Protect DoS)
-	$(DOCKER_CMD) $(PLUS_ARGS) --secret id=rhel_license,src=rhel_license --build-arg BUILD_OS=ubi-plus-nap --build-arg NAP_MODULES=dos
+	$(DOCKER_CMD) $(PLUS_ARGS) --secret id=rhel_license,src=rhel_license --build-arg BUILD_OS=ubi-plus-dos --build-arg NAP_MODULES=dos
```

Comment:
think this should still be `BUILD_OS=ubi-plus-nap` as both dos and waf are different NAP products

Reply:
Have updated the naming for both to make it more clear, thanks!